translations/messages.en.yaml

```diff
@@ -794,6 +794,11 @@ subscription_sort: Subscription sort
 alphabetically: Alphabetically
 subscriptions_in_own_sidebar: Subscriptions separate
 sidebars_same_side: Sidebars on the same side
```

Comment:
I think this and the above translation can be cleaned up but doesn't need to be part of this PR, maybe we can wait and see if people disagreed with me and want it back :laughing:

Reply:
you can always change translations in weblate :D

Reply:
can `en` translations be removed from the weblate side? I wasn't sure if it was the "sentinel" version and is only editable on this end. I don't want to break weblate again :grimacing: